build/charts/antrea/README.md

```diff
@@ -12,7 +12,7 @@ Kubernetes networking based on Open vSwitch
 
 ## Requirements
 
-Kubernetes: `>= 1.16.0-0`
+Kubernetes: `>= 1.19.0-0`
```

Comment:
Hi @jainpulkit22 can you submit a seperate PR for this version update? we need to backport both in here https://github.com/antrea-io/antrea/pull/6396

Reply:
sure I can, I added it here because verify docs check were failing

Reply:
@luolanzone done PR: https://github.com/antrea-io/antrea/pull/6397

Reply:
@jainpulkit22 , the changes are still visible in this PR as in #6397